src/System.Runtime.Extensions/tests/System/Environment.ProcessorCount.cs

```diff
@@ -32,11 +32,11 @@ public void Windows_ProcessorCountTest()
         public void Unix_ProcessorCountTest()
         {
             //arrange
-            int _SC_NPROCESSORS_ONLN =
-                RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? 84 :
-                RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD")) ? 1002 :
-                58;
-            int expected = (int)sysconf(_SC_NPROCESSORS_ONLN);
+            int _SC_NPROCESSORS_CONF =
+                RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? 83 :
+                RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD")) ? 1001 :
+                57;
+            int expected = (int)sysconf(_SC_NPROCESSORS_CONF);
```

Comment:
I assume you're going to update this to match the other release/2.1 PR?

Reply:
@stephentoub Right, the changes in this PR will be altered to be exactly the same as in release/2.1 corefx PR